### PR TITLE
Add command optional flags explanation on Wiki

### LINKED ISSUE #26: Add command optional flags explanation on Wiki.
## Why
`Create` and `run test` commands depend on the template on a default git branch but we can specify the branch that we want to use as a template by adding flags.
 
## Who Benefits?
Gin template users

### PR DESCRIPTION
**Pull request by gutakk**
## What happened 👀
Resolve #26 
- Add instruction on [wiki](https://github.com/nimblehq/gin-templates/wiki/Commands)
- Update README
 
## Insight 📝
N/A

## Proof Of Work 📹
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Our templates offer a rich boilerplate to jump start Go Gin-based application de
   go build -o $GOPATH/bin/nimble-gin github.com/nimblehq/gin-templates
   ```
 
-- Create the project using gin-templates. Note that the **main** branch is used.
+- Create the project using gin-templates. Note that the **main** branch is used. Refer to [this wiki page](https://github.com/nimblehq/gin-templates/wiki/Commands) for instructions on how to use a different branch.
```
Comment (junan) [inline README.md]: Can we change the text like this?
```suggestion
- Create the project using gin-templates. Note that the **main** branch is being used by default. Refer to [this wiki page](https://github.com/nimblehq/gin-templates/wiki/Commands) for instructions on how to use a different branch.
```

File: README.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Our templates offer a rich boilerplate to jump start Go Gin-based application de
 
 ## Test
 
-Execute all unit tests:
+- Execute all unit tests. Note that the **main** branch is used. Refer to [this wiki page](https://github.com/nimblehq/gin-templates/wiki/Commands) for instructions on how to use a different branch.
```
Comment (junan) [inline README.md]: Same as previous comment

File: README.md (hunk 1 above)
Comment (gutakk) [reply README.md]: Good catch. Fixed in 65a9b78c2557ed0af301851210e84459452c4c1b

File: README.md (hunk 2 above)
Comment (gutakk) [reply README.md]: Good catch. Fixed in 65a9b78c2557ed0af301851210e84459452c4c1b

**Review by junan — APPROVED** [review]

**Review by olivierobert — CHANGES_REQUESTED** [review]
Be mindful of English syntax (Grammarly is your friend):

image

Prefer the following:

The template uses Cookiecutter to generate a new application project. By default, the `main` branch is used. However, a different branch, just as `develop` - can be used.

**Comment by gutakk** [comment]
> The template uses Cookiecutter to generate a new application project. By default, the `main` branch is used. However, a different branch, just as `develop` - can be used.

Fixed https://github.com/nimblehq/gin-templates/wiki/Commands.

I used the free version of Grammarly to correct the grammar and seems like my phrase doesn't break any of their rules. Not sure if Grammarly can adjust the phrase same as you suggest?

**Comment by olivierobert** [comment]
@gutakk Note that you can also use our education allowance to subscribe to a paid version of Grammarly.

**Review by olivierobert — APPROVED** [review]

**Pull request merged**